db/schema.rb

```diff
@@ -0,0 +1,63 @@
+# encoding: UTF-8
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# Note that this schema.rb definition is the authoritative source for your
+# database schema. If you need to create the application database on another
+# system, you should be using db:schema:load, not running all the migrations
+# from scratch. The latter is a flawed and unsustainable approach (the more migrations
+# you'll amass, the slower it'll run and the greater likelihood for issues).
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema.define(version: 20160427162835) do
+
+  create_table "markets", force: :cascade do |t|
+    t.string   "name"
+    t.string   "address"
+    t.string   "city"
+    t.string   "county"
+    t.string   "state"
+    t.string   "zip"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.integer  "fake_id"
+  end
+
+  create_table "products", force: :cascade do |t|
+    t.string   "name"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.integer  "fake_id"
+    t.integer  "vendor_id"
+  end
+
+  add_index "products", ["vendor_id"], name: "index_products_on_vendor_id"
+
+  create_table "sales", force: :cascade do |t|
+    t.string   "name"
+    t.integer  "amount"
+    t.datetime "purchase_time"
+    t.datetime "created_at",    null: false
+    t.datetime "updated_at",    null: false
+    t.integer  "fake_id"
+    t.integer  "vendor_id"
+    t.integer  "product_id"
+  end
+
+  add_index "sales", ["product_id"], name: "index_sales_on_product_id"
+  add_index "sales", ["vendor_id"], name: "index_sales_on_vendor_id"
+
+  create_table "vendors", force: :cascade do |t|
+    t.string   "name"
+    t.string   "no_of_employees"
+    t.datetime "created_at",      null: false
+    t.datetime "updated_at",      null: false
+    t.integer  "fake_id"
+    t.integer  "market_id"
+  end
+
+  add_index "vendors", ["market_id"], name: "index_vendors_on_market_id"
+
+end
```

Comment:
`fake_id` isn't a great variable name - it's not descriptive of what your data is